go-controller/pkg/node/management-port_linux_test.go

```diff
@@ -321,7 +321,7 @@ func testManagementPortDPU(ctx *cli.Context, fexec *ovntest.FakeExec, testNS ns.
 	_, err = config.InitConfig(ctx, fexec, nil)
 	Expect(err).NotTo(HaveOccurred())
 
-	nodeAnnotator := kube.NewNodeAnnotator(&kube.KubeOVN{kube.Kube{fakeClient}, egressipv1fake.NewSimpleClientset(), &egressfirewallfake.Clientset{}, nil}, existingNode.Name)
```

Comment:
oh since we now pass named arguments, we don't need to pass empty values/nils anymore for cloudnetworkclient etc since if the key is missing its auto-understood?